dev/opeyemiferanmi1/helper.py

```diff
@@ -0,0 +1,69 @@
+#import libraries
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split, cross_val_score, GridSearchCV
+from sklearn.svm import SVC
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import StratifiedKFold
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.naive_bayes import GaussianNB
+from sklearn.metrics import classification_report, confusion_matrix, accuracy_score
+import pandas as pd
+
+def data_description(data):
+    """Displays some basic descriptions of the imported data"""
+    print(pd.DataFrame(data.head().T))
+    print("\n")
+    print('Shape of the data: ' + str(data.shape) + "\n")
+    print("\n")
+    print('Data description',(pd.DataFrame(data.describe())))
+    print("\n")
+    print('Vehicle unique Classes', data.Class.unique(), "\n")
+
+def scale_fit_transform_data(df, column_list):
+    """Takes in a dataframe and a list of column names to transform
+        returns a dataframe of scaled values"""
+    df_to_scale = df[column_list]
+    x = df_to_scale.values
+    scaler = StandardScaler()
+    x_scaled = scaler.fit_transform(x)
+    df_to_scale = pd.DataFrame(x_scaled, columns=df_to_scale.columns)
+    return df_to_scale
+
+def spot_check_algorithm(x_train, y_train, n_splits=10 ):
```

Comment:
What is the motivation for specifying a 10 splits?
Please justify this choice in comments or int he notebook.

Reply:
I have added this justification to the notebook